packages/elements-dev-portal/src/containers/StoplightProject.tsx

```diff
@@ -129,7 +129,9 @@ const StoplightProjectImpl: React.FC<StoplightProjectProps> = ({
     elem = (
       <NodeContent
         node={node}
-        Link={Link}
+        Link={({ children, ...props }) => {
+          return <a href={props.to}>{children}</a>;
+        }}
```

Comment:
Neither the original code nor this `<a>` tag works correctly with scrolling. (@jgreen44 so maybe we'll revert this specific code and go back to the `react-router-dom` component.

From https://knowbody.github.io/react-router-docs/api/Link.html:
> Note: React Router currently does not manage scroll position, and will not scroll to the element corresponding to the hash. Scroll position management utilities are available in the scroll-behavior library.

The hashes are in the URL bar in Jason's PR when running locally now, but they don't scroll when clicked. We assume this is because the browser tries to resolve the anchor before the content is actually loaded, but I don't know how to fix that. Do we need to do something special to accomplish this? https://v5.reactrouter.com/web/guides/scroll-restoration/generic-solution

Reply:
Yeah, it seems the problem is unrelated se we might get back to router link. But the scrolling itself is tricky. Could we window.scroll it to the header?

Reply:
I just messaged Jason this but figured I'd put it here for others to sanity-check me:

- Jason and I found this SO thread yesterday: https://stackoverflow.com/a/45777157
- It mentions a third party dep `react-router-hash-link`, and we were both like “we don’t want to import another dep just for this”.
- I checked, and it looks like we already pull that into elements: https://github.com/stoplightio/elements/blob/0069c4648f109fee8485e94aa999a66699bcdac1/packages/elements-core/package.json#L74
- We also apparently use it here: https://github.com/stoplightio/elements/blob/0069c4648f109fee8485e94aa999a66699bcdac1/packages/elements-core/src/components/MarkdownViewer/CustomComponents/ReactRouterLink.tsx#L3
- So I see we have this component `ReactRouterMarkdownLink ` which uses the HashLink thing from `react-router-hash-link`, which supposedly has some magic sauce that takes care of scrolling.
- So… I wonder if we can use that `ReactRouterMarkdownLink` component instead of the vanilla `Link` from `react-router-dom` here: https://github.com/stoplightio/elements/blob/0069c4648f109fee8485e94aa999a66699bcdac1/packages/elements-dev-portal/src/containers/StoplightProject.tsx#L132

Reply:
Perfect, that's it! I didn't even remember that we had it there. We should be able to use that. I've just quickly checked if that helps and looks like scrolling to anchors works fine.

Reply:
@mallachari : Pushed latest commit with @Nezteb 's suggestion. I cannot yalc into Platform without errors. If someone could look at this code to see if it'll work.

Reply:
Sure, I'll take a look

Reply:
@mallachari : Were you actually able to get the scrolling to the anchors working?